scripts/performance.ps1

```diff
@@ -143,31 +149,42 @@ function Run-Loopback-Test() {
     Stop-Background-Executable -Process $proc
 
     $MedianCurrentResult = Median-Test-Results -FullResults $allRunsResults
+    Write-Host "Current Run: $MedianCurrentResult kbps"
 
-    $fullLastResult = Get-Latest-Test-Results -Platform $Platform -Test "loopback"
-    $MedianLastResult = 0
-    if ($fullLastResult -ne "") {
-        $MedianLastResult = Median-Test-Results -FullResults $fullLastResult.individualRunResults
-    }
-
-    $ToPublishResults = [TestPublishResult]::new()
-
-    $ToPublishResults.CommitHash = $CurrentCommitHash.Substring(0, 7)
-    $ToPublishResults.PlatformName = $Platform
-    $ToPublishResults.TestName = "loopback"
-    $ToPublishResults.IndividualRunResults = $allRunsResults
-
-    $ResultsFolderRoot = "$Platform/loopback"
-    $ResultsFileName = "/results.json"
-
-    $NewFilePath = Join-Path $RootDir "artifacts/PerfDataResults/$ResultsFolderRoot"
-    $NewFileLocation = Join-Path $NewFilePath $ResultsFileName
-    New-Item $NewFilePath -ItemType Directory -Force
+    if ($PGO) {
+        $Artifacts = Join-Path $RootDir "\artifacts\windows\$($Arch)_$($Config)_$($Tls)"
```

Comment:
Should we be doing this after Run-Loopback-Test is called? That way when we add more tests, they'll automatically be PGO'd too?

Reply:
Ah, You're right. I'll fix this